### PR TITLE
Fix bookworm and readme

### PR DESCRIPTION
**Pull request by bugoff**
# Description

Updated readme this cookbook only supports Chef 15.5+ 

Since Debian Bookworm the ntp user is ntpsec (i expect Debian will continue to use this in all later versions)

For Debian / redhat based os'es there is a leap-second file, prefer to use this.

## Issues Resolved
- Readme 
https://github.com/sous-chefs/ntp/issues/230

- bookworm
https://github.com/sous-chefs/ntp/issues/234
https://github.com/sous-chefs/ntp/issues/224

- leap-second file
https://github.com/sous-chefs/ntp/issues/217

## Check List

- [x] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -42,8 +42,15 @@
 default['ntp']['statsdir'] = '/var/log/ntpstats/'
 default['ntp']['conf_owner'] = 'root'
 default['ntp']['conf_group'] = 'root'
-default['ntp']['var_owner'] = 'ntp'
-default['ntp']['var_group'] = 'ntp'
+
+if platform?('debian') && node['platform_version'].to_i <= 12
```
Comment (damacus) [inline attributes/default.rb:46]:
```suggestion
if platform?('debian') && node['platform_version'].to_i < 12
```

Shouldn't this be less than? Might need more coffee

**Review by damacus — CHANGES_REQUESTED** [review]
Excellent! thank you

**Review by damacus — DISMISSED** [review]

File: kitchen.dokken.yml [hunk 2]
```diff
@@ -32,11 +27,6 @@ platforms:
       image: dokken/centos-stream-8
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: centos-stream-9
```
Comment (damacus) [inline kitchen.dokken.yml:35]: There's no point removing these files here, they're templated in. 

We should try and support stream-9 over centos-7 in the future too.

Comment (bugoff) [reply kitchen.dokken.yml:35]: i create a new issue for this, i'll do this in a separate PR
https://github.com/sous-chefs/ntp/issues/244

File: README.md [hunk 3]
```diff
@@ -218,7 +217,7 @@ These attributes are set based on platform / system information provided by Ohai
 - `ntp['leapfile']`
 
   - String, the path to the ntp leapfile.
-  - Default, /etc/ntp.leapseconds.
+  - Default, /etc/ntp.leapseconds. Default for debian: /usr/share/zoneinfo/leap-seconds.list, for redhat: /usr/share/zoneinfo/leapseconds
```
[the remark this answers is not in the document]
Comment (bugoff) [reply README.md:220]: any suggestions on how to format it?=

Comment (damacus) [inline README.md:220]:
```suggestion
  - Default: `/etc/ntp.leapseconds` 
  - Debian default: `/usr/share/zoneinfo/leap-seconds.list`, 
  - RedHat default: `/usr/share/zoneinfo/leapseconds`
```

**Comment by kitchen-porter** [comment]
Released as: [5.1.0](https://github.com/sous-chefs/ntp/releases/tag/5.1.0)

**Comment by damacus** [comment]
Thankas @bugoff! that's been released ❤️

**Pull request merged**